Controller/RoleController.php

```diff
@@ -116,8 +117,15 @@ public function createRoleAction(Request $request)
      */
     public function updateRoleAction(Request $request, $roleId)
```

Comment:
> And contenttype also uses "update", unless you have renamed it recently :)

Mmm actually I missed that you were duplicating what's done in `createRoleAction()`. You probably need to move the new creation logic to that action and let `NotFound` exception bubble up.